README.md

```diff
@@ -34,8 +34,10 @@ This repository contains source for Sustainable Education Foundation Website. [V
 #### Want to contribute to this project? 
 
 **Send us a Pull Request**
-
```

Comment:
I think it's better to have this line (empty line after `**Send us a Pull Request**`).

Reply:
@Piumal1999 actually I removed it because @anjula-sack asked me to. can I know for sure what to do , whether to leave the space in line 38 or remove it.

Reply:
@Piumal1999 I made the requested changes. Please take a look and make sure if that's the change required .

Reply:
@Piumal1999 Extremely sorry there's been a misunderstanding @anjula-sack requested the space removal in another PR in the same line 38.  anyway requested changes are made. please take a look.

Reply:
It's alright. No worries🤗